Topics/Tech_Stacks.md

```diff
@@ -1,7 +1,8 @@
 ## Tech Stacks
 
 ### [Learning MySQL for databases](./Tech_Stacks/Learning_MySQL.md)
+### [Nuxt3](https://github.com/IbrahimBess/learning-software-engineering.github.io/blob/patch-2/Topics/Tech_Stacks/Nuxt3.md)
```

Comment:
@IbrahimBess My bad, I tested this and it should not link to the github file, but rather the page itself e.g., `(./Tech_Stacks/Nuxt3.md)`

Can you create a new PR to fix this? Thank you